From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] Disable QtScriptBindings check
Forwarded; not-needed
Origin: vendor
Last-Update: 2011-07-06

QtScriptBindings is a runtime dependency which is not needed at build time.
Just make cmake think that QtScriptBindings is available.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -90,8 +90,8 @@ if( WITH_PLAYER )
     macro_ensure_version( "4.5.0" ${KDE_VERSION} KDE_HAS_KIMAGECACHE )
     macro_bool_to_01( KDE_HAS_KIMAGECACHE HAVE_KIMAGECACHE )
 
-    find_package( QtScriptQtBindings )
-    macro_log_feature( QTSCRIPTQTBINDINGS_FOUND "qtscript-qt" "QtScript Qt Bindings" "http://code.google.com/p/qtscriptgenerator/" FALSE "" "" )
+    # Package level dependencies ensure that QtScriptBindings exist at runtime
+    set(QTSCRIPTQTBINDINGS_FOUND TRUE CACHE BOOL "Just assume QtScriptbindings exist [ Debian-specific ]")
 
     add_definitions(${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
 
